eva-accession-release-automation/publish_release_to_ftp/publish_release_files_to_ftp.py

```diff
@@ -168,44 +169,49 @@ def hardlink_to_previous_release_assembly_files_in_ftp(current_release_assembly_
-                                        .format(public_current_release_assembly_folder,
-                                                public_previous_release_assembly_folder)
-                                        , 'ln -f {0} {1}'.format(file_to_hardlink,
-                                                                 public_current_release_assembly_folder))
+                run_command_with_output(f"""Creating hardlink from previous release assembly folder 
+                                        {public_current_release_assembly_folder} to current release assembly folder 
+                                        {public_previous_release_assembly_folder}""",
+                                        f'ln -f {file_to_hardlink} {public_current_release_assembly_folder}')
     else:
         raise Exception("Previous release folder {0} does not exist for assembly!"
                         .format(public_previous_release_assembly_folder))
 
 
-def publish_assembly_release_files_to_ftp(current_release_assembly_info, release_properties):
+def publish_assembly_release_files_to_ftp(current_release_assembly_info, release_properties,
+                                          species_current_release_folder_name):
     assembly_accession = current_release_assembly_info["assembly_accession"]
-    public_release_assembly_folder = \
-        get_folder_path_for_assembly(release_properties.public_ftp_current_release_folder, assembly_accession)
+    public_release_assembly_folder = get_folder_path_for_assembly(release_properties.public_ftp_current_release_folder,
+                                                                  assembly_accession)
+    public_release_species_assembly_folder = \
+        get_folder_path_for_species_assembly(release_properties.public_ftp_current_release_folder,
+                                             species_current_release_folder_name, assembly_accession)
     # If a species was processed during this release, copy current release data to FTP
     if current_release_assembly_info["should_be_released"] and \
             current_release_assembly_info["num_rs_to_release"] > 0:
         copy_current_assembly_data_to_ftp(current_release_assembly_info, release_properties,
-                                          public_release_assembly_folder)
-    else:
-        # Since the assembly data is unchanged from the last release, hard-link instead of symlink to older release data
-        # so that deleting data in older releases does not impact the newer releases
-        # (hard-linking preserves the underlying data for a link until all links to that data are deleted)
-        hardlink_to_previous_release_assembly_files_in_ftp(current_release_assembly_info, release_properties)
+                                          public_release_assembly_folder, public_release_species_assembly_folder)
+    # Below code to hardlink to previous release commented since we will be releasing everything with new regime
```

Comment:
I wonder if we need to modify how we load the release tracker for this release, to make sure we grab everything... another ticket I guess.

Reply:
Agree. We need to make sure, we load the tracker in a way all the species are picked up.
Will create a ticket, so that we remember to check it

Reply:
I guess this will probably be covered in [EVA-3325](https://www.ebi.ac.uk/panda/jira/browse/EVA-3325).